sdk/core/core-http/lib/policies/proxyPolicy.ts

```diff
@@ -26,6 +26,14 @@ function loadEnvironmentProxyValue(): string | undefined {
     return process.env[Constants.HTTP_PROXY];
   } else if (process.env[Constants.HTTP_PROXY.toLowerCase()]) {
     return process.env[Constants.HTTP_PROXY.toLowerCase()];
+  } else if (process.env[Constants.NO_PROXY]) {
+    return process.env[Constants.NO_PROXY];
```

Comment:
`NO_PROXY` is used to specify a lists of hosts that should be exempt from using the proxy I thought, rather than being a proxy URL itself.